.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Is there a reason why we only want to run CI on the main branch? I think it's nice when pushing to a branch to see if tests pass before I open a PR

Reply:
Yeah I think that makes sense